dask_geomodeling/raster/elemwise.py

```diff
@@ -730,9 +730,12 @@ def process(*args):
                 data_list.append(data["values"])
                 no_data_values.append(data["no_data_value"])
 
-        dtype = np.result_type(*data_list)
+        dtype = args[0]['dtype']
         fillvalue = get_dtype_max(dtype)
 
+        if not data_list:
+            return {"values": fillvalue, "no_data_value": fillvalue}
```

Comment:
This is the case if all of the sources return `None`. In that case, `FillNoData` should also return `None`.